functions/index.js

```diff
@@ -0,0 +1,45 @@
+const functions = require("firebase-functions");
+const express = require("express");
+const admin = require("firebase-admin");
+
+// init express server
+const app = express();
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+// init firestore connection
+admin.initializeApp();
+const db = admin.firestore();
```

Comment:
Maybe we need to keep an eye on this declaration, since in the firebase.js file we declare those lines:
export const auth = firebase.auth;
export const db = firebase.database();
export const firestore = firebase.firestore();
Might cause some confusion